### DISCUSSION TITLE
F# language version

### DISCUSSION THREAD

**Discussion by ihordyrman** [codewars/codewars.com#2532] [Q&A] [answered]
Is it possible to update the current version of F# on CodeWars?
Version 6.0 is already available and have a lot of nice features that makes code much cleaner and readable. 
Would be extremely nice to have these features while doing katas. 🙏

**Comment by hobovsky** [comment, marked as answer]
There's this request for code runner: https://github.com/codewars/runner/issues/97 which you can upvote.
